devel/bitkeeper/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                bitkeeper
+version             7.3.1ce
+categories          devel
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+maintainers         {l2dy @l2dy} openmaintainer
+description         BitKeeper is a fast, enterprise-ready, distributed SCM \
+                    that scales up to very large projects and down to tiny ones.
+long_description    ${description}
+homepage            http://www.bitkeeper.org/
+master_sites        http://www.bitkeeper.org/downloads/${version}/
+distname            bk-${version}
+distfiles           ${distname}.src${extract.suffix}
+
+checksums           rmd160  a1f75be06beec3bb36a5436c00c90e3caee7aab3 \
+                    sha256  8784fe01722ebac7d93854c25a3aa203ec2751d90d06b036003868f2d8e6d250
+
+depends_build       port:pkgconfig
+
+depends_lib         port:libtomcrypt \
+                    port:libtommath \
+                    port:lz4 \
+                    port:pcre \
+                    port:zlib
+
+patchfiles          patch-src-conf.mk.local.diff \
+                    patch-src-gui-tcltk-Makefile.diff
+
+post-patch {
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/src/conf.mk.local
+}
+
+use_configure       no
+
+build.args-append   CC=${configure.cc} \
+                    CXX=${configure.cxx} \
+                    CPP=${configure.cpp}
```

Comment:
How should I use it? `configure.cxx_stdlib libstdc++`?
I don't have <10.9 system and can't test it.

    - Support clang's -stdlib option explicitly using configure.cxx_stdlib
      rather than relying on its default value. This can be used to make
      MacPorts use libc++ on OS X < 10.9. (larryv in r110519)